clang/include/clang/Basic/AttrDocs.td

```diff
@@ -3545,6 +3545,29 @@ scope) variables and static local variable as well.
   }];
 }
 
+def OpenCLUSMAddressSpacesDocs : Documentation {
+  let Category = DocOpenCLAddressSpaces;
+  let Heading = "[[clang::opencl_usm_device]], [[clang::opencl_usm_host]]";
+  let Content = [{
+The (usm_device) and (usm_host) address space attributes specify that an object
+is allocated in global memory on the device/host. It helps distinguishing USM
+pointers that access device memory and accessors that access global memory from
+those that access host memory.
+  }];
+}
+
+def OpenCLAddressSpaceGlobalHostDocs : Documentation {
```

Comment:
You should be able to delete this section, since it is unused now.

Reply:
Oops, missed this one